storybook/cypress/e2e/legend-item.cy.ts

```diff
@@ -1,6 +1,7 @@
 describe("Legend Item", () => {
   beforeEach(() => {
     cy.visit("http://localhost:45000/iframe.html?id=core-legend-item--default");
+    cy.injectAxe();
```

Comment:
Waarom wordt Axe zowel in de beforeEach() geinjected, als in de diverse tests?

Reply:
Da's goed gespot. In de `beforeEach()` is `injectAxe()` nu weg. Verder gehandhaaft na elke `visit()`